expo/expo
# [eyw] Add engines limitation for Node 14 and above

# Why

Replacement PR for #12731, closes #12674

This package was built for internal usage at Expo. We do allow other people to use it, but we won't support older Node LTS versions which are soon-to-be either EOL or in maintenance mode.

- Node 12 was LTS until `2020-11-30`, where it was put into maintenance mode.
- Node 14 was LTS from `2020-11-30`, and will be put into maintenance mode from `2021-10-19`

(see [Node Releases](https://nodejs.org/en/about/releases/))

# How

Added `engines` definition to the package json.

# Test Plan

- Use Node <14
- Check if an engine error/warning is provided

# Checklist
